packs/napalm/config.yaml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
`config.schema.yaml` was introduced in v1.5 (https://stackstorm.com/2016/06/30/improved-pack-configuration-user-scoped-datastore-items-secure-secrets-store/). 

Both versions were supported for the last ~6 months or so. But the new pack management `st2 pack config` command will only work with `config.schema.yaml` (https://docs.stackstorm.com/latest/packs.html#configuring-a-pack). It is still technically possible to use older style configuration with new packs, but it is less user-friendly, and should be discouraged.

There should be very few remaining packs using `config.yaml`. I've made many changes over the last few weeks to migrate packs. As of yesterday, around 12 of 105 packs were using `config.yaml`. Probably fewer by now.